src/order_gui.cpp

```diff
@@ -1166,7 +1166,11 @@ struct OrdersWindow : public Window {
 			case WID_O_DEPOT_ACTION: {
 				VehicleOrderID sel = this->OrderGetSel();
 				const Order *order = this->vehicle->GetOrder(sel);
-				if (order == nullptr || !order->IsType(OT_GOTO_DEPOT)) break;
+				if (order == nullptr || !order->IsType(OT_GOTO_DEPOT)) {
+					/* We can't leave this param empty or the undefined behavior can cause a crash. */
+					SetDParam(0, STR_EMPTY);
```

Comment:
Maybe say "We can't leave this param unset", it reads a bit strange to that you may not leave it empty and yet it is set to STR_EMPTY :)

Reply:
Indeed. 🙂